hadoop-hdds/container-service/src/main/java/org/apache/hadoop/ozone/container/ozoneimpl/ContainerReader.java

```diff
@@ -207,6 +210,14 @@ public void verifyAndFixupContainerData(ContainerData containerData)
         KeyValueContainerUtil.parseKVContainerData(kvContainerData, config);
         KeyValueContainer kvContainer = new KeyValueContainer(kvContainerData,
             config);
+        if (kvContainer.getContainerState() == RECOVERING) {
+          if (shouldDeleteRecovering) {
+            kvContainer.delete();
```

Comment:
Nit: Typo: delete --> Delete

Reply:
Also I assume this delete will take care of cleaning the rocksDB entries as well.
CC: @guihecheng

Reply:
yea, we should also delete the rocksDB entry of this container in merge rocksdb case

Reply:
I think `kvContainer.delete()` already handles deleting kv pairs in the rocksdb for both before and after rocksdb-merge.